# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/coreutils-misc.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Not available in some quick avr32 testbuild.

  - Rene Rebe <rene@exactcode.de>

--- coreutils-6.12/lib/c-strtod.c.vanilla	2008-11-06 15:26:32.000000000 +0100
+++ coreutils-6.12/lib/c-strtod.c	2008-11-06 15:28:07.000000000 +0100
@@ -37,7 +37,7 @@
 #endif
 
 /* c_strtold falls back on strtod if strtold doesn't conform to C99.  */
-#if LONG && HAVE_C99_STRTOLD
+#if LONG && HAVE_C99_STRTOLD && !defined(__UCLIBC__)
 # define STRTOD strtold
 #else
 # define STRTOD strtod
@@ -48,7 +48,7 @@
 {
   DOUBLE r;
 
-#ifdef LC_ALL_MASK
+#if defined(LC_ALL_MASK) && !defined(__UCLIBC__)
 
   locale_t c_locale = newlocale (LC_ALL_MASK, "C", 0);
   r = STRTOD_L (nptr, endptr, c_locale);
